### PR TITLE
add forks to the job details

### PR DESCRIPTION
**Pull request by jainnikhil30**
##### SUMMARY
This PR adds the forks on the Job Details page like we had in Ansible Tower

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION

After the PR:
forks

```

```

### CODE REVIEW

File: awx/ui/src/screens/Job/JobDetail/JobDetail.js [hunk 1]
```diff
@@ -361,6 +361,14 @@ function JobDetail({ job, inventorySourceLabels }) {
             value={t`True`}
           />
         )}
+        {typeof job.forks === 'number' &&
+          typeof job.forks === 'number' && (
```
Comment (keithjgrant) [inline awx/ui/src/screens/Job/JobDetail/JobDetail.js]:
```suggestion
          (
```

**Review by nixocio — APPROVED** [review]
Thanks @jainnikhil30

File: awx/ui/src/screens/Job/JobDetail/JobDetail.js [hunk 2]
```diff
@@ -361,6 +361,14 @@ function JobDetail({ job, inventorySourceLabels }) {
...
           />
         )}
+        {typeof job.forks === 'number' &&
+          (
+            <Detail
+              dataCy="forks"
+              label={t`Forks`}
+              value={`${job.forks}`}
+            />
+        )}
```
Comment (keithjgrant) [inline awx/ui/src/screens/Job/JobDetail/JobDetail.js]:
```suggestion
        {typeof job.forks === 'number' && (
          <Detail dataCy="forks" label={t`Forks`} value={`${job.forks}`} />
        )}
```

Comment (keithjgrant) [reply awx/ui/src/screens/Job/JobDetail/JobDetail.js]: looks like suggested edit made the format checking unhappy, sorry 😅 Let's see if this one is better

Comment (kialam) [reply awx/ui/src/screens/Job/JobDetail/JobDetail.js]: We will also want to add a popover tooltip for this detail item I think. That can be done as a separate PR too.

**Comment by kialam** [comment]
We will also want to add a popover tooltip for this detail item. That can be done as a separate PR.

**Comment by jay-steurer** [comment]
Tested and approved. Cypress test automation updated.

**Pull request merged**